src/toplevelprops.ts

```diff
@@ -57,25 +57,25 @@ export interface AutoSizeParams {
   contains?: 'content' | 'padding';
 }
 
-export function normalizeAutoSize(autosize: AutosizeType | AutoSizeParams, isUnitOrLayer: boolean): AutoSizeParams {
-  if (!autosize) {
-    return {
-      type: 'pad'
-    };
-  }
-  const autoSizeParams: AutoSizeParams = {
+function normalizeAutoSize(autosize: AutosizeType | AutoSizeParams) {
+  return isString(autosize) ? {type: autosize} : autosize || {};
+}
+
+export function normalizeAutoSizes(topLevelAutosize: AutosizeType | AutoSizeParams, configAutosize: AutosizeType | AutoSizeParams, isUnitOrLayer: boolean = true): AutoSizeParams {
```

Comment:
The plural form of `normalizeAutoSizes` reads a bit funny to me, but I guess that's fine.

Reply:
Wonder if we should rename this to `normalizeAutoSize` (since this one is a public API, its name is more important) and rename the other to `normalizeSingleAutoSize` or `_normalizeAutoSize` or something.

Reply:
Done